model/src/main/proto/interaction_object.proto

```diff
@@ -95,6 +95,8 @@ message ImageWithRegions {
     }
 
     Region region = 2;
+    // The description for the region.
```

Comment:
For consistency, please add an empty newline above this documentation comment.

Reply:
done